apps/src/templates/sectionsRefresh/SingleSectionSetUp.jsx

```diff
@@ -20,6 +27,17 @@ export default function SingleSectionSetUp({
           onChange={e => updateSection('name', e.target.value)}
         />
       </label>
+      <MultiSelectGroup
+        label="Grade (choose at least one)"
+        name="grades"
+        required={true}
+        options={options}
+        values={grades}
+        setValues={g => {
+          updateSection('grades', g);
+          setGrades(g);
```

Comment:
Why do you need to `getGrades` here?

Reply:
notes from meeting: made some changes to try to address this (6cd1679) but i'm still storing the `values` in two different ways, which seems unnecessary. not confident that i'm doing the right things inside these components yet